src/shared/components/ProfilePage/StatsCategory/styles.scss

```diff
@@ -0,0 +1,235 @@
+  background-color: $tc-white;
+}
+
+@mixin module-l {
+  @include module-s;
+
+  max-width: 1242px;
+  margin-left: 10px;
+  margin-right: 10px;
+  padding-top: 30px;
+
+  @media only screen and (min-width: 900px) {
+    padding-top: 30px;
+  }
+}
+
+.categories {
+  @include module-l;
+
+  margin-left: auto;
+  margin-right: auto;
+  padding-top: 15px;
+  align-self: center;
+  background: $tc-white;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+
+  .track {
+    margin-top: 22.4px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding-bottom: 30px;
+
+    &.noclick {
+      cursor: default;
+    }
+
+    .name {
+      margin: auto;
+      font-size: 20px;
+      line-height: 24px;
+      margin-bottom: 10.6px;
+
+      @include sofia-pro-extra-light;
+    }
+
+    svg {
+      height: 20px;
+      width: 20px;
+      margin-right: 7px;
+      margin-bottom: -2px;
+    }
+
+    .subtrack {
+      text-decoration: none;
+      color: $tc-black;
+      display: flex;
+      flex-direction: row;
+      justify-content: space-between;
+      align-items: center;
+      width: 300px;
+      height: 52px;
+      border-bottom: 1px solid $tc-gray-neutral-dark;
+
+      &.first {
+        border-top: 1px solid $tc-gray-neutral-dark;
+      }
+
+      &:hover {
+        background-color: $tc-gray-neutral-dark;
+      }
+
+      .name {
+        margin-left: 15px;
+        align-self: center;
+        flex-basis: 250px;
+        text-align: left;
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+        font-size: 14px;
+        line-height: 18px;
+        text-transform: uppercase;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin-top: 12px;
+
+        @include sofia-pro-light;
+      }
+
+      .ranking {
+        margin: auto;
+        margin-right: 15px;
+        color: $tc-gray-80;
+        display: flex;
+        flex-direction: column;
+        align-items: flex-end;
+
+        @include sofia-pro-medium;
+
+        .number {
+          text-align: center;
+          font-size: 13.4px;
+          line-height: 16px;
+
+          span.square {
+            position: absolute;
+            margin-left: 5px;
+            width: 6px;
+            height: 6px;
+          }
+        }
+
+        .tag {
+          text-transform: uppercase;
+          margin: 5px auto 0 auto;
+
+          @include sofia-pro-regular;
+
+          font-size: 12px;
+          line-height: 10px;
+          color: $tc-gray-60;
+        }
+      }
+
+      .arrow {
+        @include sofia-pro-extra-light;
+
+        color: $tc-gray-80;
+        height: 12px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+      }
+    }
+
+    .icon {
+      width: 80px;
+      height: 80px;
+      margin: auto;
+    }
+  }
+}
+
+@media (min-width: 768px) {
```

Comment:
Actually, here and in many other places, we should use our standard media mixins from `topcoder-react-utils`:
https://github.com/topcoder-platform/topcoder-react-utils/blob/master/docs/scss-mixins.md#media-mixins
e.g. in this very place it should be
```scss
@include xs-to-sm {
  // styling
}
```

also we prefer to have small media mixins in each style class, rather than a few huge media blocks that completely overwrite all styles.